app.js

```diff
@@ -138,42 +65,85 @@ function numToTime(num) {
   }
 }
 
-/** prints an unordered list of the hourly sale simulations to the page **/
-function printStoreSalesSimsList(store) {
+/** adds the header row of times to the given table based on the hours of a given Store **/
+function renderSalesTableHead(table, store) {
+  var headRow = document.createElement('tr');
+  headRow.setAttribute('class', 'head-row');
+
+  headRow.appendChild(document.createElement('th')); //empty cell
+
+  var hourCell;
+  for (var i = 0; i < (store.closeAt - store.openAt); i++) {
+    hourCell = document.createElement('th');
+    hourCell.textContent = numToTime(store.openAt + i);
+    headRow.appendChild(hourCell);
+  }
+  var totCookiesCell = document.createElement('th');
+  totCookiesCell.textContent = 'Daily Location Total';
+  headRow.appendChild(totCookiesCell);
+
+  table.appendChild(headRow);
+}
+
+/** adds the footer row of totals to the given table based on an array of Stores **/
+function renderTotalsFoot(table, stores) {
+  var footRow = document.createElement('tr');
+  footRow.setAttribute('class', 'totals-row');
+
+  var totalCell = document.createElement('td');
+  totalCell.setAttribute('class', 'tot-cell');
+  totalCell.textContent = 'Totals';
+  footRow.appendChild(totalCell);
+
+  var modelStore = stores[0];
+
+  var totCookiesCell;
+  var totForHour;
+  var totForDay = 0;
+  for (var i = 0; i < modelStore.cookiesHrs.length; i++) {
```

Comment:
instead of your modelStore.cookieHrs.length, consider the same format used above to generate the cookieHrs:
```JavaScript
for (var i = 0; i < (store.closeAt - store.openAt); i++) {
```